src/main/navigation/navigation.c

```diff
@@ -2400,14 +2422,14 @@ static navigationFSMEvent_t navOnEnteringState_NAV_STATE_FW_LANDING_GLIDE(naviga
         return NAV_FSM_EVENT_SWITCH_TO_NAV_STATE_FW_LANDING_ABORT;
     }
 
-    if (getLandAltitude() <= posControl.fwLandState.landAltAgl + navFwAutolandConfig()->flareAltitude) {
+    if (getHwRangefinderStatus() == HW_SENSOR_OK && getLandAltitude() <= posControl.fwLandState.landAltAgl + navFwAutolandConfig()->flareAltitude) {
```

Comment:
`posControl.flags.estAglStatus >= EST_USABLE` might be better here ?

Reply:
If then `posControl.flags.estAglStatus == EST_TRUSTED` we are very close to the ground, should only be active if the rangefinder is accurate.

Reply:
OK, I realised you're using the direct output from the range finder in `getLandAltitude `so this makes sense in that case.